### PR TITLE
Flatten Distribution class

### LINKED ISSUE #340: Distill `DistributionBounds` into `Distribution`
We currently have two classes used to describe distributions for `NWMConfig`:

```python
class DistributionBounds(Serializable):
    minimum: int = 0
    maximum: int = 0
    distribution_type: Literal["normal"] = "normal"

    class Config:
        feilds = {
            "distribution_type": {"alias": "type"},
            "minimum": {"alias": "min"},
            "maximum": {"alias": "max"},
        }


class Distribution(Serializable):
    """
    Represents the definition of a distribution of numbers
    """

    distribution: DistributionBounds

    def __init__(
        self, minimum: int = 0, maximum: int = 0, distribution_type: str = "normal"
    ):
```

All `Distribution` does is carry around a `DistributionBounds`, therefore the fields in `DistributionBounds` need to be inserted into `Distribution`. This will result in an implementation that just looks like:

```python
class Distribution(Serializable):
    """
    Represents the definition of a distribution of numbers
    """
    minimum: int = 0
    maximum: int = 0
    distribution_type: Literal["normal"] = "normal"

    def __init__(
        self, minimum: int = 0, maximum: int = 0, distribution_type: str = "normal"
    ):
```
This will reduce configurations from looking like:

```json
{
    "parameters": {
        "land_cover": {
            "distribution": {
                "minimum": 1,
                "maximum": 12,
                "distribution_type": "normal"
            }
        }
    }
}
```

to

```json
{
    "parameters": {
        "land_cover": {
            "minimum": 1,
            "maximum": 12,
            "distribution_type": "normal"
        }
    }
}
```

### PR DESCRIPTION
**Pull request by aaraney**
closes #340

## Removals

- Remove `DistributionBounds` class

## Changes

- Flatten `Distribution` class

## Checklist

- [x] PR has an informative and human-readable title
- [x] Changes are limited to a single goal (no scope creep)
- [x] Code can be automatically merged (no conflicts)
- [x] Code follows project standards (link if applicable)
- [x] Passes all existing automated tests
- [x] Any _change_ in functionality is tested
- [x] New functions are documented (with a description, list of inputs, and expected output)
- [x] Placeholder code is flagged / future todos are captured in comments
- [x] Visually tested in supported browsers and devices (see checklist below :point_down:)
- [x] Project documentation has been updated (including the "Unreleased" section of the CHANGELOG)
- [x] Reviewers requested with the [Reviewers tool](https://help.github.com/articles/requesting-a-pull-request-review/) :arrow_right:

### CODE REVIEW

File: python/lib/communication/dmod/communication/maas_request/distribution.py [hunk 1]
```diff
@@ -3,67 +3,27 @@
...
+
     maximum: int = 0
+    """The upper bound for the distribution"""
+
     distribution_type: Literal["normal"] = "normal"
+    """The type of the distribution"""
 
-    class Config:
+    class Config(Serializable.Config):
         feilds = {
```
Comment (christophertubbs) [inline python/lib/communication/dmod/communication/maas_request/distribution.py:21]: Typo or can me not raed?

**Review by christophertubbs — CHANGES_REQUESTED** [review]
I think an important field has been misspelled that can kill some alias functionality.

File: python/lib/communication/dmod/communication/maas_request/distribution.py (hunk 1 above)
Comment (aaraney) [reply python/lib/communication/dmod/communication/maas_request/distribution.py:21]: @christophertubbs, what typo? `Config` inheriting from `Serializable.Config`?

Comment (aaraney) [reply python/lib/communication/dmod/communication/maas_request/distribution.py]: what a blunder... thanks for catching this.

**Comment by aaraney** [comment]
@christophertubbs, fixed the typo. This is ready for another review.

**Pull request merged**